.github/workflows/build-and-push-docker-image.yml

```diff
@@ -27,8 +27,20 @@ jobs:
         env:
           ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
         run: |
-          docker build . -t $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA
+          image_id=$(docker build . -q -t $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA)
+          echo IMAGE_ID=$image_id >> $GITHUB_ENV
           echo "::set-output name=image::$ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA"
           docker image tag $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA $ECR_REGISTRY/$ECR_REPOSITORY:latest
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:$GITHUB_SHA
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:latest
+      - name: Copy wasm artifact from the image
+        run: |
+          container_id=$(docker create ${{ env.IMAGE_ID }})
+          docker cp $container_id:/home/cere/node-runtime-artifacts/node_runtime.compact.wasm ./
+          short_github_sha=${GITHUB_SHA:0:9}
```

Comment:
Why 9?

Reply:
I decided to stick to the length git uses (image). But now I see that the usual length is 7 even for git
<img width="381" alt="image" src="https://user-images.githubusercontent.com/21261739/183006189-fa5386e3-2f5e-4de4-92d2-31e9db7f85a8.png">